Libraries/src/Amazon.Lambda.CloudWatchEvents/ECSEvents/Attachment.cs

```diff
@@ -30,4 +30,10 @@ public class Attachment
         /// </summary>
         public string Type { get; set; }
     }
+    
+    public class NameValuePair
+    {
+        public string Name { get; set; }
+        public string Value { get; set; }
```

Comment:
this should fix your test errors in 3.1 the DefaultLambdaSerializer is picky when it comes the names since there is a mix of camelCase and PascelCase.

```suggestion
      [DataMember(Name = "name")]
#if NETCOREAPP_3_1
      [System.Text.Json.Serialization.JsonPropertyName("name")]
#endif
        public string Name { get; set; }
        
      [DataMember(Name = "value")]
#if NETCOREAPP_3_1
      [System.Text.Json.Serialization.JsonPropertyName("value")]
#endif
        public string Value { get; set; }
```